### PR TITLE
Bael 3283

### PR DESCRIPTION
**Pull request by sandy03934**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -773,6 +773,7 @@
                 <module>spring-mvc-kotlin</module>
                 <module>spring-mvc-simple</module>
                 <module>spring-mvc-simple-2</module>
+				<module>spring-mvc-views</module>
```
Comment (KevinGilmore) [inline pom.xml]: use spaces to indent

File: pom.xml [hunk 2]
```diff
@@ -1513,6 +1514,7 @@
...
```
Comment (KevinGilmore) [inline pom.xml]: use spaces to indent

File: spring-mvc-views/pom.xml [hunk 3]
```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-mvc-views</artifactId>
+    <packaging>war</packaging>
+    <name>spring-mvc-themes</name>
```
Comment (KevinGilmore) [inline spring-mvc-views/pom.xml]: name, if provided, match artifactId as per guidelines

File: spring-mvc-views/src/main/java/com/baeldung/config/SecurityConfig.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        auth.jdbcAuthentication()
+                .passwordEncoder(passwordEncoder())
+                .dataSource(dataSource);
```
Comment (KevinGilmore) [inline spring-mvc-views/src/main/java/com/baeldung/config/SecurityConfig.java]: indentation should be 4 spaces over from "auth" in line 28

File: spring-mvc-views/src/main/java/com/baeldung/config/SecurityConfig.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                .antMatchers("/anonymous*").anonymous()
+                .antMatchers("/login*").permitAll()
+                .anyRequest().authenticated()
+                .and()
+                .formLogin()
+                .and()
+                .logout()
+                .logoutUrl("/logout.do")
+                .invalidateHttpSession(true)
+                .clearAuthentication(true);
```
Comment (KevinGilmore) [inline spring-mvc-views/src/main/java/com/baeldung/config/SecurityConfig.java]: indentation should be 4 spaces over from "http" in line 36

File: spring-mvc-views/src/main/java/com/baeldung/config/DataSourceConfig.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import javax.sql.DataSource;
+
+@EnableJpaRepositories(basePackages = "com.baeldung")
+public class DataSourceConfig {
+    @Bean
+    public DataSource dataSource() {
+        EmbeddedDatabaseBuilder builder = new EmbeddedDatabaseBuilder();
+        return builder.setType(EmbeddedDatabaseType.HSQL)
+                .addScripts("db/sql/create-db.sql", "db/sql/insert-data.sql")
+                .build();
```
Comment (KevinGilmore) [inline spring-mvc-views/src/main/java/com/baeldung/config/DataSourceConfig.java]: indentation should be 4 spaces over from "return" in line 21

File: pom.xml (hunk 1 above)
Comment (sandy03934) [reply pom.xml]: Ok

File: spring-mvc-views/pom.xml (hunk 3 above)
Comment (sandy03934) [reply spring-mvc-views/pom.xml]: Sure. I missed it during refactoring. Thanks for highlighting that

**Pull request merged**